README.md

```diff
@@ -118,17 +119,17 @@ Terminology
                   produce the finished article (built conda distributions)
 
 
-Updating pvlib-python-feedstock
-===============================
+Updating pvlib-feedstock
+========================
```

Comment:
Some of these automated changes to the README are not really desirable.  Since they're just in the README, maybe we don't care?  I suspect that if we revert them the automated re-render will just make the same changes again next time anyway.

Reply:
I didn't find any relevant docs indicating yea or nay for it being a good idea to edit the README manually, and I don't think any of these changes will cause problems, just maybe be a little confusing.  So I'm inclined to leave as is.